helpers/createProposal.sh

```diff
@@ -157,6 +76,11 @@ function main {
 
     mainnet_uri=$(chain_to_uri 1)
 
+    if [[ "$chains" == "12" ]]; then
```

Comment:
should we put a higher number like 100 so we don't have to change when adding more chains